presto-main/src/main/java/com/facebook/presto/sql/planner/GroupingOperationRewriter.java

```diff
@@ -57,16 +54,9 @@ public GroupingOperationRewriter(QuerySpecification queryNode, Analysis analysis
         this.groupIdSymbol = requireNonNull(groupIdSymbol, "groupIdSymbol is null");
     }
 
-    @Override
-    public Expression rewriteExpression(Expression node, Void context, ExpressionTreeRewriter<Void> treeRewriter)
+    public Expression rewrite(GroupingOperation expression)
```

Comment:
I'll mark the class final, with a private constructor and the single method static but I won't make the method final as per https://github.com/prestodb/presto/pull/6704#discussion_r107256106